WinUIGallery/App.xaml.cs

```diff
@@ -24,6 +24,7 @@
 using System.Text;
 using Windows.System;
 using System.Runtime.InteropServices;
+using Microsoft.Toolkit.Uwp.Notifications;
```

Comment:
Would you be able to use the WinAppSDK [Notifications](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/app-notifications/app-notifications-quickstart?tabs=cs) instead from the Community Toolkit?

Reply:
I like using the community toolkit more so I used it.

Reply:
Can it be like this?    
using Microsoft.Windows.AppNotifications;
using Microsoft.Windows.AppNotifications.Builder;
       //Create the notification.
            var notification = new AppNotificationBuilder()
                .AddText("An exception was thrown.")
                .AddText($"Type: {e.Exception.GetType()}")
                .AddText($"Message: {e.Message}")
                .AddText($"HResult: {e.Exception.HResult}")
                .BuildNotification();

            //Show the notification
            AppNotificationManager.Default.Show(notification);

Reply:
Changed it.